specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-03-01/capacityReservation.json

```diff
@@ -361,6 +361,22 @@
               "name": "ExpandTypesForGetCapacityReservationGroups",
               "modelAsString": true
             }
+          },
+          {
+            "name": "resourceIdsOnly",
```

Comment:
It's recommended to use odata to filter data: https://github.com/microsoft/api-guidelines/blob/194adeb29bdb958712ae89f863aed35d8469df21/graph/articles/collections.md?plain=1#L162